remoteworkitem/trackeritem_repository.go

```diff
@@ -62,7 +62,7 @@ func lookupIdentities(ctx context.Context, db *gorm.DB, remoteWorkItem RemoteWor
 	identityRepository := account.NewIdentityRepository(db)
 	//spaceSelfURL := rest.AbsoluteURL(goa.ContextRequest(ctx), app.SpaceHref(spaceID.String()))
 	workItem := workitem.WorkItem{
-		ID:      remoteWorkItem.ID,
+		// ID:      remoteWorkItem.ID,
```

Comment:
well, what if an issue with the same number already exists before importing GH issues ?